modules/codegen/src/main/scala/com/twilio/guardrail/generators/Java/DropwizardGenerator.scala

```diff
@@ -10,7 +10,7 @@ import com.twilio.guardrail.terms.framework._
 object DropwizardGenerator {
   object FrameworkInterp extends (FrameworkTerm[JavaLanguage, ?] ~> Target) {
     def apply[T](term: FrameworkTerm[JavaLanguage, T]): Target[T] = term match {
-      case FileType(format)   => safeParseType(format.getOrElse("java.io.File"))
+      case FileType(format)   => safeParseType(format.getOrElse("java.io.InputStream"))
```

Comment:
so a concern here is what happens if there are more than one `type: file` parameters?

Reply:
Shouldn't be a problem, I don't think.  Why would it be?  They still get unique parameter names.  No reason the caller can't provide more than one.

Reply:
`curl example -F file1=@file1.iso -F file2=@file2.iso` and `curl example -F file2=@file2.iso -F file1=@file1.iso` are _very_ different.

Reply:
Not sure I understand what you think the issue is, or how using InputStream instead of File changes anything about this.

Reply:
Actually, even having two input streams overcomplicates the problem. It's simpler to just have one:
```scala
def doFoo(a: Boolean, b: InputStream)
```
So, this is fine:
1. `curl http://... -F a=false -F b=@file.iso`
1. `a` is a valid `Boolean`
1. `b` is streamed directly to `InputStream`, unbuffered
1. Everyone's happy

... but this would be not great
1. `curl http://... -F b=@file.iso -F a=false`
1. `b` is streamed directly to `InputStream`, buffered?
1. `a` may be a valid boolean, but we have no way to know yet
1. `doFoo` can't be called yet, as `a` has not been supplied yet and no mutable variables
1. server runs out of memory

Reply:
Oh, you're talking about server, not client.

Yeah, that's an issue, to be sure.  But this is also why servers have global max bytes per request settings.

Reply:
Oh, yes, very much so.

Reply:
So, we're OK with buffering in ram for arbitrary file sizes up until the global max bytes, and if anyone wants different behaviour it's up to them to ensure that only file content is supplied via post body?

Reply:
I'm not sure, honestly.  I don't think `File` is ideal, because allowing the server to spool out data to a temporary file comes with its own problems.  Buffering in RAM is also not great, but at least the user has control over the max body size allowed.  Multipart is just kinda crappy from a streaming perspective, unfortunately.

Reply:
It occurs to me that spec writers can also override this behavior with a `x-jvm-type: java.io.File` if they want.  I _do_ think that `InputStream` is a better default, though.

Reply:
What's the behaviour if they do override it with `java.io.File`, `mktemp()`? Sounds dangerous

Reply:
Not sure, will have to read the Jersey source to figure that out, but I assume it uses https://docs.oracle.com/javase/7/docs/api/java/io/File.html#createTempFile(java.lang.String,%20java.lang.String,%20java.io.File)

If they do this, then they should be aware of the risks.  This is the current behavior, anyway.